src/promptflow/promptflow/_sdk/_submitter/experiment_orchestrator.py

```diff
@@ -166,6 +168,14 @@ def _test_flow_node(self, node, test_context):
     def _test_command_node(self, *args, **kwargs):
         raise NotImplementedError
 
+    def _test_chat_group_node(self, node, test_context):
+        from promptflow._sdk.entities._chat_group._chat_group import ChatGroup
+
+        chat_group = ChatGroup._from_node(node, test_context)
+        logger.debug(f"Invoking chat group node {node.name!r}.")
+        chat_group.invoke()
```

Comment:
what's the behavior of invoke a chat_group multiple times?

Reply:
no need to take action in this PR

Reply:
Good catch, the `conversation_history` will have cached records and that's the only impact. It's a one-line change and I have added in this pr.